callback-heaven.lisp

```diff
@@ -268,15 +268,26 @@ Note that this memory is not further managed!"
               (write-string " {" stream)
               (terpri stream)
               (format stream "~4T")
-              (unless (eq :void (api-function-return-type f))
-                (write-string "return " stream))
-              (format stream "((~A)(~A[~D]))(~{~A~^, ~});"
+              (format stream "~A ret;" (type-name-to-foreign (api-function-return-type f)))
+              (terpri stream)
```

Comment:
Why not `~%` at the previous `format` instead of `terpri`?

Reply:
Just following the convention that was used in the rest of the file. I will break with tradition, however.